README.md

```diff
@@ -9,9 +9,6 @@ To update `pants`
 
     brew update && brew upgrade pants
 
-> Notice:
```

Comment:
I'd probably leave a variant of this notice - because there are enough ways to break these updates. 

> If installed via Brew, it is recommended to update the `pants` binary via `brew update && brew upgrade`. Do not use the `SCIE_BOOT=update pants` mechanism, as that will replace the Brew symlink with a copy of the downloaded binary. 

> This script makes a best-effort attempt to overwrite that downloaded binary on `brew upgrade`, however it's best to avoid this problem all together.

^ ?

Reply:
There's only a single file involved, as long as the preflight hack here works, I don't see the need for such a notice.
If it breaks in other ways, it's likely caused by something else we didn't cover in the notice any way.  

I'm open to revisit the question if we find more caveats though.. or it turns out to be a real thing to watch out for. Sounds reasonable?